scheduler/task.go

```diff
@@ -42,6 +42,57 @@ func createTaskInfo(task types.EremeticTask, offer *mesos.Offer) (types.Eremetic
 		})
 	}
 
+	var portResources []*mesos.Value_Range
+	var portMapping   []*mesos.ContainerInfo_DockerInfo_PortMapping
+
+	if(len(task.Ports) > 0) {
```

Comment:
Of course a helper function that finds contiguous sequences in a list would do...